[2025-04-09T08:25:15.979+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-09T08:25:16.024+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: init_elt_pipeline.load_data_from_minio_to_duckdb manual__2025-04-09T08:25:10.899501+00:00 [queued]>
[2025-04-09T08:25:16.035+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: init_elt_pipeline.load_data_from_minio_to_duckdb manual__2025-04-09T08:25:10.899501+00:00 [queued]>
[2025-04-09T08:25:16.037+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-04-09T08:25:16.056+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_data_from_minio_to_duckdb> on 2025-04-09 08:25:10.899501+00:00
[2025-04-09T08:25:16.064+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=486) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-09T08:25:16.066+0000] {standard_task_runner.py:63} INFO - Started process 498 to run task
[2025-04-09T08:25:16.064+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'init_elt_pipeline', 'load_data_from_minio_to_duckdb', 'manual__2025-04-09T08:25:10.899501+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/init_elt_pipeline.py', '--cfg-path', '/tmp/tmpv50zo4aa']
[2025-04-09T08:25:16.067+0000] {standard_task_runner.py:91} INFO - Job 5: Subtask load_data_from_minio_to_duckdb
[2025-04-09T08:25:16.088+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-09T08:25:16.125+0000] {task_command.py:426} INFO - Running <TaskInstance: init_elt_pipeline.load_data_from_minio_to_duckdb manual__2025-04-09T08:25:10.899501+00:00 [running]> on host 208e59669cb5
[2025-04-09T08:25:16.225+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='init_elt_pipeline' AIRFLOW_CTX_TASK_ID='load_data_from_minio_to_duckdb' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T08:25:10.899501+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-09T08:25:10.899501+00:00'
[2025-04-09T08:25:16.228+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-09T08:25:16.257+0000] {init_minio.py:24} INFO - Minio client connected successfully
[2025-04-09T08:25:16.283+0000] {extractor.py:137} ERROR - Error loading data to DuckDB: S3 operation failed; code: NoSuchKey, message: Object does not exist, resource: /bitcoin/bitcoin-2021-01-01, request_id: 183498F0EFD3869C, host_id: dd9025bab4ad464b049177c95eb6ebf374d3b3fd1af9251148b658df7ac2e3e8, bucket_name: bitcoin, object_name: bitcoin-2021-01-01
[2025-04-09T08:25:16.284+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-09T08:25:16.284+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-09T08:25:16.322+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=init_elt_pipeline, task_id=load_data_from_minio_to_duckdb, run_id=manual__2025-04-09T08:25:10.899501+00:00, execution_date=20250409T082510, start_date=20250409T082516, end_date=20250409T082516
[2025-04-09T08:25:16.401+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-09T08:25:16.422+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-09T08:25:16.427+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
